packages/iov-bns/src/types.ts

```diff
@@ -214,6 +214,18 @@ export interface AddAddressToUsernameTx extends UnsignedTransaction {
   readonly payload: ChainAddressPair;
 }
 
+export interface Participant {
+  readonly signature: Uint8Array;
```

Comment:
Why the hack is this field called signature in the backend but constans an address?

Can we expose `readonly address: Address` here and hide all the strangeness?

Reply:
Yep we can, the name comes from weave, it just means more conversion functions and less casting.

Reply:
Please use proper names in the weave types. Then raise issues, so we improve the protobuf naming.